### PR TITLE
Translation and cache maps improvements

### PR DESCRIPTION
**Pull request by andrixnet**
Many translation improvements on english strings. 
Some corrections in newcache, editcache and viewcache pages and repaired hardcoded strings.
Improved configurability of static map and minimap in viewcache page.

### CODE REVIEW

**Comment by andrixnet** [comment]
English strings are not complete, some may be marked with ***, this is a work in progress.

**Comment by andrixnet** [comment]
Changes can be seen live at OC RO for the duration of this weekend, I will resync to master on monday.

**Comment by andrixnet** [comment]
Adds short URL for OC waypoint in cache page, as discussed here https://github.com/opencaching/opencaching-pl/issues/424

**Comment by andrixnet** [comment]
Fixes static map layer and zoom level https://github.com/opencaching/opencaching-pl/issues/410 point 1)
Fixes external map links https://github.com/opencaching/opencaching-pl/issues/410 point 2)
Dirty fix for highlighting current cache on minimap https://github.com/opencaching/opencaching-pl/issues/410 point 3.5)

**Comment by kojoty** [comment]
@andrixnet, - to be sure - we discussed last days the issues around committing pull requests and there is an idea **to not to commit anything what is not absolutely necessary between Friday and Monday (I mean during the weekends).**

We would like to avoid disturbing the users in their weekend geocache trips with our "well-enough-tested" (especially mine) commits...

**Comment by deg-pl** [comment]
not:
'hint_instructions' => 'Używaj tylko litery z alfabetu angielskiego i / lub cyfr. <br /> Wszystko napisane w ciągu <code>[nawiasach kwadratowych]</code> zostaną pokazane odkodowane.',
but
'hint_instructions' => 'Używaj tylko litery z alfabetu angielskiego i / lub cyfr. <br /> Wszystko napisane w ciągu <code>[nawiasach kwadratowych]</code> zostanie pokazane odkodowane.',
(zostaną -> zostanie)

**Comment by andrixnet** [comment]
@kojoty I perfectly agree about not disturbing users during the weekend. 
My understanding is that this refers to commits to origin/master , which get deployed on production servers. Please correct me if I am wrong.
**Be sure that I will not attempt to merge anything during the weekends.**

Also, I personally will not attempt to merge any changes in the code, but respectfully request review.
I may try to merge myself translation only and non-code modifications.

There is no urgency to this pull request, I submitted it after I finished working and testing on these changes so far (let's say I completed a stage). And for review. 

Thank you.

**Comment by andrixnet** [comment]
@harrieklomp corrected typos as indicated
@deg-pl corrected polish string

Question on github usage _(I haven't been able to figure out so far)_: can I add newer commits to this pull request or do I have to delete it and create a new one?

**Comment by deg-pl** [comment]
I think you should add new commit to this pull.
PLEASE - DON'T MERGE THIS PULL BEFORE ALL NODES HAVE DONE CONFIGURATION CHANGES ($config['maps']...)

**Comment by andrixnet** [comment]
Please check the diffs for several new NL and PL strings. I added them with Google translate to give an idea of their function.

**Comment by andrixnet** [comment]
@deg-pl you can withdraw the "Don't commit" status. It is not necessary.

All new configuration variables were introduced in settingsDefault.inc.php (so they apply by default) and their value set to behave like before these enhancements. 
(with only one exception: Flopp's map will appear, my bad)

Explicit changes in settings.inc.php are **ONLY** necessary if a node desires to change the default behaviour.

What you see on OC RO at this time is however a result of customization in settings.inc.php based on these new variables. It is set on my translation branch intentionally to show and allow review.

**Comment by deg-pl** [comment]
Yes, you're right. Done ;)

**Comment by andrixnet** [comment]
I just set to on all external maps on OC RO, so that one can see how they work. I also did a small improvement / correction to Szukacz URL.

**Comment by deg-pl** [comment]
Very good!
To do:
ro

**Comment by andrixnet** [comment]
I presume it may be "b.d." as it was hardcoded originally, but I wanted to make sure.

**Comment by andrixnet** [comment]
Added oc_waypoint to external maps, needed by Flopp's Map, according to https://github.com/flopp/FloppsTolleKarte/issues/57#issuecomment-204700286

File: lib/languages/pl.php [hunk 1]
```diff
@@ -318,7 +320,8 @@
     'optional' => 'opcjonalnie',
     'time_distance_hint' => 'Czas i odległość zależą często od warunków zewnętrznych, więc jeśli nie jesteś w stanie oszacować tych danych, wpisz w powyższe pola zera. Więcej informacji znajdziesz w artykule <a href="{wiki_link_cacheParams}" target="_BLANK">"czas i droga"</a>. Dane te przydatne są na przykład przy skrzynkach typu <b>multicache</b> i <b>wydarzenie</b>.',
     'select_one' => 'proszę wybrać!',
-    'waypoint_other_portal_info' => 'Waypoint z innych baz danych dla tej skrzynki. Żadne dane nie są pobierane automatycznie.',
+    'foreign_waypoint' => 'Waypointy obce',
+    'foreign_waypoint_info' => 'Waypoint z innych baz danych dla tej skrzynki. Żadne dane nie są pobierane automatycznie.',
```
Comment (andrixnet) [inline lib/languages/pl.php]: please check these 2 lines

File: lib/languages/pl.php [hunk 2]
```diff
@@ -351,7 +354,8 @@
     'no_javascript' => 'Obsługa Javascript jest obecnie wyłączona w Twojej przeglądarce. Możesz wprowadzać tylko zwykły tekst. Aby wprawdzić kod HTML za pomocą edytora, musisz włączyć obsługe Javascript.',
     'html_usage' => 'Proszę używać znaczników HTML wyłącznie do formatowania tekstu.',
     'available_html' => 'Wykaz dostępnych znaczników HTML',
-    'additional_enc_info' => '<b><font color="red">Dodatkowe</font> informacje kodowane.</b> <i>Pole nieobowiązkowe</i>. Tutaj możesz umieścić dodatkowe informacje ułatwiające znalezienie skrzynki. Informacje bez których znalezienie skrzynki nie będzie możliwe, muszą znaleźć się w polu "Pełen opis".',
+    'hint_long_text' => '<b><font color="red">Dodatkowe</font> informacje kodowane.</b> <i>Pole nieobowiązkowe</i>. Tutaj możesz umieścić dodatkowe informacje ułatwiające znalezienie skrzynki. Informacje bez których znalezienie skrzynki nie będzie możliwe, muszą znaleźć się w polu "Pełen opis".',
+    'hint_instructions' => 'Używaj tylko litery z alfabetu angielskiego i / lub cyfr. <br /> Wszystko napisane w ciągu <code>[nawiasach kwadratowych]</code> zostanie pokazane odkodowane.',
```
Comment (andrixnet) [inline lib/languages/pl.php]: please check these 2 lines

File: lib/languages/pl.php [hunk 3]
```diff
@@ -2239,18 +2237,29 @@
...
+    'contact_nl_desc_5_2' => 'Because the volunteers at <i>opencaching.nl</i> do not always have time to answer your questions. So do not be impatient if there is not an immediate answer to a question.',
+    'contact_nl_title_6' => 'Donate',
+    'contact_nl_desc_6_1' => 'As you can see, you have the opportunity to donate to Opencaching Benelux This would be for the organization a huge help and support and will be appreciated. More info on our <a href="http://forum.opencaching.nl/viewforum.php?f=27"> forum </a> and/or our <a href="http://wiki.opencaching.nl/index.php/Financieel">wiki. </a>',
+    'contact_nl_desc_6_2' => '<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top"><input type="hidden" name="cmd" value="_s-xclick"><input type="hidden" name="hosted_button_id" value="2QNGQ2WN6A44E"><input type="image" src="https://www.paypalobjects.com/nl_NL/BE/i/btn/btn_donateCC_LG.gif" border="0" name="submit" alt="PayPal, de veilige en complete manier van online betalen."><img alt="" border="0" src="https://www.paypalobjects.com/nl_NL/i/scr/pixel.gif" width="1" height="1"></form>',
```
Comment (andrixnet) [inline lib/languages/pl.php]: Please tranlsate these

File: lib/languages/nl.php [hunk 4]
```diff
@@ -318,7 +320,8 @@
     'optional' => 'optioneel',
     'time_distance_hint' => 'Tijd en afstand zijn soms door externe faktoren moelijk in te schatten. Kun je geen schatting maken van inspanning vul dan in één of beide velden een 0 in. Dit soort gegevens zijn belangrijk voor caches zoals <b>multicaches</b> of <b>evenementen</b>.',
     'select_one' => 'selecteer!',
-    'waypoint_other_portal_info' => 'Waypoint voor deze cache bevind zich in andere databases. Gegevens worden niet overgenomen.',
+    'foreign_waypoint' => 'Vreemde waypoints',
+    'foreign_waypoint_info' => 'Waypoint voor deze cache bevind zich in andere databases. Gegevens worden niet overgenomen.',
```
Comment (andrixnet) [inline lib/languages/nl.php]: Please check these

File: lib/languages/nl.php [hunk 5]
```diff
@@ -351,7 +354,8 @@
     'no_javascript' => 'Je Browser ondersteunt nog geen Javascript. Je kunt alleen gewone tekst gebruiken.Om de HTML editor te gebruiken die je Javascript in te schakelen.',
     'html_usage' => 'Gebruik HTML tags voor het bewerken van tekst.',
     'available_html' => 'Beschikbare HTML tags lijst',
-    'additional_enc_info' => '<b><font color="red">Extra </font> versleutelde hint.</b> <i>Geen verplicht veld</i>. Hier kun een een extra hint toevoegen. Informatie om de cache normaal te kunnen vinden dient in de "Omschrijving" beschreven te staan.',
+    'hint_long_text' => '<b><span style="color:red">Extra</span> versleutelde hint.</b> <i>Geen verplicht veld</i>. Hier kun een een extra hint toevoegen. Informatie om de cache normaal te kunnen vinden dient in de "Omschrijving" beschreven te staan.',
+    'hint_instructions' => 'Gebruik alleen letters van het Engels alfabet en / of cijfers. <br /> Alles wat geschreven in <code>[vierkante haken]</code> wordt weergegeven ongecodeerd.',
```
Comment (andrixnet) [inline lib/languages/nl.php]: Please check these

File: lib/languages/pl.php (hunk 1 above)
Comment (deg-pl) [inline lib/languages/pl.php]: OK

File: lib/languages/pl.php (hunk 2 above)
Comment (deg-pl) [inline lib/languages/pl.php]: Remove word "ciągu".
Correct: Wszystko napisane w [nawiasach...

File: lib/languages/nl.php (hunk 4 above)
Comment (harrieklomp) [inline lib/languages/nl.php]: Is correct

File: lib/languages/nl.php (hunk 5 above)
Comment (harrieklomp) [inline lib/languages/nl.php]: Is correct

File: lib/languages/pl.php (hunk 2 above)
Comment (andrixnet) [inline lib/languages/pl.php]: corrected.

File: lib/languages/pl.php (hunk 3 above)
Comment (andrixnet) [inline lib/languages/pl.php]: Not urgent, for the merge, I believe.

**Comment by andrixnet** [comment]
New / changed strings should be ok now. I believe it is ready for merge.

**Pull request merged**